.github/workflows/aws-prod.yaml

```diff
@@ -12,17 +12,17 @@ jobs:
     environment: production
     env:
       AWS_REGION: us-east-1
-      CLUSTER_NAME: chainlog-prod
+      CLUSTER_NAME: maker-prod
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Configure AWS credentials
-      uses: aws-actions/configure-aws-credentials@v1
+      uses: aws-actions/configure-aws-credentials@v2
       with:
-        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-access-key-id: ${{ secrets.EKS_AWS_ACCESS_KEY_ID }}
```

Comment:
I'd call these `PROD_AWS_ACCESS_KEY_ID` and `STAGING_AWS_ACCESS_KEY_ID` for prod and staging respectively. As they're used for more than EKS

Reply:
@eskp done, I am using Github Actions environments here so secret names don't need to have STAGING_ or PROD_ prefixes